main_test.go

```diff
@@ -50,7 +55,14 @@ func TestGoogleOauth(t *testing.T) {
 		t.Error(err)
 	}
 
-	mux := NewMux(muxLog, googleClient, nopZoomClient)
+	zat := &Config{
+		logger:       muxLog,
+		copies:       make(map[int64]string, 0),
```

Comment:
This is the new required param?

Reply:
Looks like the tests don't need `copies` so I'll take that out. The main change seems to be that `NewMux` now takes a `Config` rather than separate logger/client params.